src/algorithms/planargradient.jl

```diff
@@ -0,0 +1,13 @@
+struct PlanarGradient <: NeutralLandscapeMaker
+    direction::Float64
+    PlanarGradient(x::T) where {T <: Real} = new(mod(x, 360.0))
+end
+
+PlanarGradient() = PlanarGradient(360rand())
+
+function landscape(alg::PlanarGradient, s::Tuple{IT,IT}=(10,30)) where {T <: Real, IT <: Integer}
+    eastness = sin(deg2rad(alg.direction))
+    southness = -1cos(deg2rad(alg.direction))
+    l = southness .* (1:s[1]) .+ (eastness .* (1:s[2]))'
+    return l
```

Comment:
I see you moved out the masking - I agree with that. The user should be able to mask manually with a mask function, no need to have that as a keyword arg.

Reply:
so... the only issue is that the scaling is done *after* the masking, which tends to break the flow of a lot of things

Reply:
my current solution would be to mask if a mask argument is given (or a series of coordinates, maybe), and then *also* give a mask function after that

Reply:
right, my mistake :+1: